tidb-cloud/release-notes-2022.md

```diff
@@ -8,6 +8,33 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2022.
 
+## November 8, 2022
+
+* Support point-in-time recovery (PITR).
```

Comment:
Please highlight that this change is for the Dedicated Tier clusters.

Reply:
Done